developer.bat

```diff
@@ -0,0 +1,17 @@
+IF NOT EXIST build mkdir build
```

Comment:
developer.bat - I think it would be better to call it something like `make_sln.bat` or `generate_sln.bat`

Reply:
I agree. It's nostalgia: my previous company had it named that way. I also like the idea of a developer 🦇

Reply:
generate_sln sounds good.